teamsupport/services.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from demands import HTTPServiceClient
+from lxml import etree
+
+from teamsupport.utils import to_xml
+
+
+class XMLHTTPServiceClient(HTTPServiceClient):
+    def _format_xml_request(self, request_params):
+        if request_params.get('send_as_xml') and (
+                request_params.get('data') or request_params.get('cdata')):
+            data = to_xml(
+                request_params['root'],
+                request_params.get('data'),
+                request_params.get('cdata'))
+            request_params['data'] = etree.tostring(
+                data, encoding='utf-8', xml_declaration=True,
+                pretty_print=True)
+            request_params.setdefault('headers', {})['Content-Type'] = (
+                'application/xml'
+            )
+        return request_params
+
+    def pre_send(self, request_params):
+        """Override this method to modify sent request parameters"""
+        request_params = super(XMLHTTPServiceClient, self).pre_send(
+            request_params)
+        return self._format_xml_request(request_params)
+
+    def parse_xml_response(self, response):
+        return etree.fromstring(response.content)
+
+
+class TeamSupportService(XMLHTTPServiceClient):
+    def __init__(self, org_id, auth_token, url=None, **kwargs):
+        url = url or 'https://app.teamsupport.com/api/xml/'
```

Comment:
If they had a sandbox or something it'd be useful, but while they don't, I'll drop it.